LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2019 Peter Evans
```

Comment:
Peter Evans? 😁

Oh, and 2019. Wasn't that looong, looooong time ago, like a life time ago?

Reply:
Please point to a preferred licence.

Reply:
I'm fine with MIT. Or with [ISC](https://github.com/gitgitgadget/gitgitgadget/blob/main/LICENSE). Just change the name and the year ;-)